proto/pulumi/language.proto

```diff
@@ -26,6 +26,8 @@ option go_package = "github.com/pulumi/pulumi/sdk/v3/proto/go;pulumirpc";
 // LanguageRuntime is the interface that the planning monitor uses to drive execution of an interpreter responsible
 // for confguring and creating resource objects.
 service LanguageRuntime {
+    rpc Handshake(LanguageHandshakeRequest) returns (LanguageHandshakeResponse) {}
```

Comment:
Would be nice to document this. In general I'd like us to build a habit of documenting public things well -- the protocols we own being a particularly important example IMO.